api/src/main/java/org/openmrs/scheduler/Schedule.java

```diff
@@ -231,21 +226,6 @@ public void setRepeatInterval(long repeatInterval) {
 	}
 	
 	/**
-	 * Get the date format used to set the start time.
-	 */
-	public String getDateFormat() {
```

Comment:
Since it was not used, i do not think any module is using it for any meaningful work. And even if any module used it, it would just remove its usage without any effects.
So no need for deprecation. 😊